### DISCUSSION TITLE
Xcode Swift error

### DISCUSSION THREAD

**Discussion by fabioz23** [NiklasMerz/cordova-plugin-fingerprint-aio#299] [Q&A]
Hi all,

anyone has this error using 4.0.1 version of this plugin?

https://github.com/NiklasMerz/cordova-plugin-fingerprint-aio/issues/295

**Comment by fabioz23** [comment]
no one?

**Comment by NiklasMerz** [comment]
Have you tried deleting the iOS platform and adding it again? Also check the Swift version in Xcode.

**Reply by fabioz23** [reply]
yes. no changes.

On Xcode workspace in build settings i see Swift 5.
